.github/workflows/rav1e.yml

```diff
@@ -273,8 +273,8 @@ jobs:
            toolchain: stable
          - conf: cargo-test
            toolchain: stable
-         - conf: cargo-c
-           toolchain: stable-x86_64-pc-windows-gnu
+#         - conf: cargo-c
```

Comment:
Please mention https://github.com/rust-lang/rust/issues/68518 and https://github.com/rust-lang/rust/pull/67429

Or we could set the `LIBRARY_PATH` to sidetrack the problem.

Reply:
Setting `LIBRARY_PATH` does not work. I would build cargo-c with the msvc toolchain and run it for the gnu toolchain.